experiments/Manifest.toml

```diff
@@ -229,7 +229,7 @@ version = "0.3.5"
 deps = ["ArtifactWrappers", "CFTime", "ClimaComms", "ClimaCore", "ClimaCoreTempestRemap", "Dates", "DiffEqCallbacks", "DocStringExtensions", "IntervalSets", "JLD2", "LinearAlgebra", "NCDatasets", "StaticArrays", "SurfaceFluxes", "Thermodynamics", "UnPack"]
 path = ".."
 uuid = "7884a58f-fab6-4fd0-82bb-ecfedb2d8430"
-version = "0.2.6"
+version = "0.2.7"
```

Comment:
why does this depend on a version of ClimaLSM instead of the local working version?

Reply:
I think it somehow does both.... like, the version is 0.2.7, but the path is specified as ".." so it looks there instead of grabbing from registry/wherever that would be stored locally?

I had just noticed that when I check the status in experiments, I see 0.2.6 ".."